.github/workflows/ubuntu.yml

```diff
@@ -166,33 +162,34 @@ jobs:
       - name: configure
         run: |
           cd _build
-          ../configure --enable-code-coverage CPPFLAGS="-Werror=declaration-after-statement"
+          ../configure --enable-code-coverage \
+                       CPPFLAGS="-Werror=declaration-after-statement -pedantic" \
```

Comment:
I suggest that we drop `-pedantic` in the CI for now (while I still want to fix a bunch of those soon, see my extra branch if you are interested; note: if this is used then GnuCOBOL limits the string size in several places, which _could_ also make a performance difference).

Reply:
I've also temporarily switched to `-std=99` as otherwise C++ comments raise errors  (even without `-pedantic`). I don't think there's an easy way to change that into a warning (*cf* https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80529).

Reply:
Ok I've reverted back to `-std=c89`.